chains/evm/deployment/v1_6_4/changesets/erc20_lockbox_deploy.go

```diff
@@ -33,9 +34,18 @@ func erc20LockboxDeployApply(mcmsRegistry *changesets.MCMSReaderRegistry) func(c
 		ds := datastore.NewMemoryDataStore()
 
 		for _, perChainInput := range input.ChainInputs {
+			tokenAdminRegistryAddress, err := datastore_utils.FindAndFormatRef(e.DataStore, datastore.AddressRef{
+				Type:          datastore.ContractType(token_admin_registry.ContractType),
+				Version:       token_admin_registry.Version,
+				ChainSelector: perChainInput.ChainSelector,
+			}, perChainInput.ChainSelector, evm_datastore_utils.ToEVMAddress)
+			if err != nil {
+				return cldf.ChangesetOutput{}, err
+			}
+
 			sequenceInput := sequences.ERC20LockboxDeploySequenceInput{
 				ChainSelector:      perChainInput.ChainSelector,
-				TokenAdminRegistry: perChainInput.TokenAdminRegistry,
+				TokenAdminRegistry: tokenAdminRegistryAddress,
 			}
 			report, err := cldf_ops.ExecuteSequence(e.OperationsBundle, sequences.ERC20LockboxDeploySequence, e.BlockChains, sequenceInput)
```

Comment:
Also, make sure that when you deploy something like a lock box or a token pool, you add the entry to the datastore with the qualifier field populated (i.e. set to the token symbol). Ensures that you can differentiate between different token pools / lock boxes in the datastore and what their purposes are.